hawc/apps/mgmt/urls.py

```diff
@@ -27,4 +27,17 @@
         views.TaskDetail.as_view(),
         name="assessment_tasks",
     ),
+    # task htmx ViewSet
```

Comment:
you'll need to delete the "api/" route in this file, and all the imports related to it, since the api.py file was deleted